modules/distribution/src/repository/resources/dbscripts/apimgt/oracle.sql

```diff
@@ -534,15 +536,15 @@ CREATE OR REPLACE TRIGGER IDP_CLAIM_TRIG
                END;
 /
 CREATE TABLE IDP_CLAIM_MAPPING (
-            ID INTEGER,
-            IDP_CLAIM_ID INTEGER,
-            TENANT_ID INTEGER,
-            LOCAL_CLAIM VARCHAR(253),
-            DEFAULT_VALUE VARCHAR(255),
-            IS_REQUESTED VARCHAR(128) DEFAULT '0',
-            PRIMARY KEY (ID),
-            UNIQUE (IDP_CLAIM_ID, TENANT_ID, LOCAL_CLAIM),
-            FOREIGN KEY (IDP_CLAIM_ID) REFERENCES IDP_CLAIM(ID) ON DELETE CASCADE)
+			ID INTEGER,
+			IDP_CLAIM_ID INTEGER,
+			TENANT_ID INTEGER,
+			LOCAL_CLAIM VARCHAR(253),
+		    DEFAULT_VALUE VARCHAR(255),
+	    	IS_REQUESTED VARCHAR(128) DEFAULT '0',
+			PRIMARY KEY (ID),
+			UNIQUE (IDP_CLAIM_ID, TENANT_ID, LOCAL_CLAIM),
+			FOREIGN KEY (IDP_CLAIM_ID) REFERENCES IDP_CLAIM(ID) ON DELETE CASCADE)
```

Comment:
Directly copied content from https://github.com/wso2/carbon-apimgt/tree/v5.0.4/features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/sql as apim version upgraded to v5.04